source/base/Text.ooc

```diff
@@ -134,6 +134,13 @@ Text: cover {
 	free: func@ ~withCriteria (criteria: Owner) -> Bool {
 		this _buffer free(criteria)
 	}
+	slice: func ~untilEnd (start: Int) -> This {
+		result := This new(this _buffer slice(start))
+		if (this _buffer owner == Owner Receiver)
+			result = result copy() // TODO: Could we be smarter here?
```

Comment:
Yeah, I think maybe the default value is only evaluated once, and not every time you call the function. So you probably can't do it like that.